internal/api/courses/users/list_test.go

```diff
@@ -0,0 +1,79 @@
+package users
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/db"
+	"github.com/edulinq/autograder/internal/model"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+func TestList(test *testing.T) {
```

Comment:
Add a test case for an empty course.
You would have to empty out a course and use a server admin to do it.
I would just make another function for it (since it doesn't fit with the pattern here).